src/Lexer.php

```diff
@@ -27,7 +27,17 @@
  *
  * @final since 4.15.0
  *
- * @phpstan-type TToken array{type: string, line: int, value: string|null, deferred: bool, tags?: list<string>, keyword?: string, keyword_type?: string, text?: string, indent?: int, columns?: list<string>}
+ * @phpstan-type TStepKeywordsType 'Given'|'When'|'Then'|'And'|'But'
+ * @phpstan-type TGeneralKeywordsType 'Feature'|'Background'|'Scenario'|'Outline'|'Examples'|'Step'
+ * @phpstan-type TKeywordsType TGeneralKeywordsType|TStepKeywordsType
```

Comment:
is this type actually used anywhere ?

Reply:
On an unrelated note, I don't like the naming of `TGeneralKeywordsType`.
The best alternative I can find is `TNonStepKeywordsType` (I thought about "TopLevel", but technically only "Feature" is top level).

Reply:
I agree this type is not worth it, especially as its only usage is on a deprecated method (I would even keep it as simply `string` in that case as it is deprecated)

Reply:
`TTitleKeywordType` sounds good to me!